src/modules/item/utils.ts

```diff
@@ -756,3 +762,19 @@ export const buildItemMappings = (mapping: Mapping, contract: LinkedContract): M
     }
   }
 }
+
+export const isWearableFileSizeValid = (fileSize: number): boolean => {
+  return fileSize > MAX_WEARABLE_FILE_SIZE
```

Comment:
shouldn't this be `<`? 

Maybe change the name to invalid? I see that you are using it that if it returns true then its invalid

Reply:
you're right! fixing it

Reply:
fixed 🙌